templates/aggkit/aggkit-config.toml

```
@@ -479,12 +479,12 @@ DBPath = "{{.zkevm_path_rw_data}}claimsponsor.sqlite"
 # ------------------------------------------------------------------------------
 # Enabled indicates if the sponsor should be run or not
 # ------------------------------------------------------------------------------
-Enabled = false
+Enabled = true
```

```suggestion
Enabled = false
```

Since having two claimers (zkevm-bridge-service claimtxmanager and aggkit claimsponsor) haven't been explicitly tested enough, maybe we could set this to `false` until we are sure we won't see unexpected behaviors

---

the new claimsponsor (which runs inside aggkit) is simply a REST API for sending claim transactions, I don’t foresee any issues running it in parallel with the legacy zkevm-bridge-service claimtxmanager. 

wdyt?